src/main/java/org/logstash/plugins/inputs/http/MessageProcessor.java

```diff
@@ -88,9 +91,8 @@ private FullHttpResponse generateResponse(Map<String, String> stringHeaders) {
         response.headers().set(headers);
 
         if (responseStatus != HttpResponseStatus.NO_CONTENT) {
-            final ByteBuf payload = Unpooled.wrappedBuffer("ok".getBytes(charset));
+            final ByteBuf payload = Unpooled.wrappedBuffer(this.responseBody.getBytes(charset));
             response.headers().set(HttpHeaderNames.CONTENT_LENGTH, payload.readableBytes());
-            response.headers().set(HttpHeaderNames.CONTENT_TYPE, "text/plain");
```

Comment:
Instead of removing the response's `content-type` we could also provide an customization option for it.

Close to `response_body` it could be also defined a new `response_content_type` which by default contains `text/plain` but in case the user could also modify it. 
In particular on registration phase, if the user customized the response body then it should be also forced to explicitly define also a response's content-type

Reply:
Yep it's a nice idea. I _hope_ I'll have a chance to resume working on this 😅